neat/channels/compilechannels

```diff
@@ -1,14 +1,27 @@
 #!/usr/bin/env python3
-import os, glob, sys, inspect, shutil, subprocess
+import imp, os, glob, sys, inspect, shutil, subprocess
```

Comment:
i'm getting a deprecation warning:
```bash
DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
```

Reply:
I've committed an implementation with the new importlib